lib/node_modules/@stdlib/stats/base/ndarray/meanpn/README.md

```diff
@@ -0,0 +1,130 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2025 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# meanpn
+
+> Compute the [arithmetic mean][arithmetic-mean] of a one-dimensional ndarray using a two-pass error correction algorithm.
+
+<section class="intro">
+
+The [arithmetic mean][arithmetic-mean] is defined as
+
+<!-- <equation class="equation" label="eq:arithmetic_mean" align="center" raw="\mu = \frac{1}{n} \sum_{i=0}^{n-1} x_i" alt="Equation for the arithmetic mean."> -->
+
+```math
+\mu = \frac{1}{n} \sum_{i=0}^{n-1} x_i
+```
+
+<!-- <div class="equation" align="center" data-raw-text="\mu = \frac{1}{n} \sum_{i=0}^{n-1} x_i" data-equation="eq:arithmetic_mean">
+    <img src="https://cdn.jsdelivr.net/gh/stdlib-js/stdlib@develop/lib/node_modules/%40stdlib/stats/base/ndarray/meanpn/docs/img/equation_arithmetic_mean.svg" alt="Equation for the arithmetic mean.">
+    <br>
+</div> -->
+
+<!-- </equation> -->
+
+</section>
+
+<!-- /.intro -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var meanpn = require( '@stdlib/stats/base/ndarray/meanpn' );
+```
+
+#### meanpn( arrays )
+
+Computes the [arithmetic mean][arithmetic-mean] of a one-dimensional ndarray using a two-pass error correction algorithm.
+
+```javascript
+var ndarray = require( '@stdlib/ndarray/base/ctor' );
+
+var xbuf = [ 1.0, 3.0, 4.0, 2.0 ];
+var x = new ndarray( 'generic', xbuf, [ 4 ], [ 1 ], 0, 'row-major' );
+
+var v = meanpn( [ x ] );
+// returns 2.5
+```
+
+The function has the following parameters:
+
+-   **arrays**: array-like object containing a one-dimensional input ndarray.
+
+</section>
+
+<!-- /.usage -->
+
+<section class="notes">
+
+## Notes
+
+-   If provided an empty one-dimensional ndarray, the function returns `NaN`.
+-   This implementation uses a two-pass approach (Neely, 1966) to compute an error correction term for improved numerical accuracy.
```

Comment:
Instead of this note, would you be willing to add the references section to this README as in https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/stats/strided/dmeanpn? Cheers!